develop/analyze-images/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Best practices for image vulnerability detection
+description: Analyze and build secure images, best practices guide
+keywords: docker scan, scan, images, vulnerability, analyze, supply chain
+redirect_from:
+  - /develop/scan-images/
+---
+
+This page contains recommendations and best practices for analyzing image
+security and building secure images.
+
+Container security spans multiple teams: developers, security, and operations.
+That's because there are multiple layers of security that apply to containers:
+
+- The container image and the software running inside
+- The interaction between a container, the host operating system, and other
+  containers on the same host
+- The host operating system
+- Container networking and storage
+
+## Analyzing images
+
+With Docker, you can bring supply chain security into the development workflow.
+The Docker platform offers an image analysis service that extends the existing,
+familiar process of vulnerability detection. Image analysis not only helps you
+detect vulnerabilities, but also reveals where each vulnerability comes from,
+and provides suggestions on how you can remediate them. The process builds on
+continuously indexing the contents of your images, and evaluating that against
+the Docker's image vulnerability database.
+
+You can trigger image analysis, view, and inspect vulnerabilities through Docker
+Hub and Docker Desktop. For detailed information, see
+[Image analysis](../../docker-hub/image-analysis.md).
+
+> **Note**
```

Comment:
Maybe move this right to the top? Consistent with how we have it on other docs pages